### PR TITLE
feat!: add Terraform 0.13 constraint and module attribution

### PR DESCRIPTION
**Pull request by bharathkkb**
This PR was auto generated, please review for accuracy before merging

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: modules/memcache/versions.tf [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      version = "~> 3.53"
+    }
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = "~> 3.53"
+    }
+  }
+
+  provider_meta "google" {
+    module_name = "blueprints/terraform/terraform-google-memorystore:memcache/v1.0.0"
```
Comment (morgante) [inline modules/memcache/versions.tf:32]:
```suggestion
    module_name = "blueprints/terraform/terraform-google-memorystore:memcache/v2.0.0"
```

File: modules/memcache/versions.tf [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  provider_meta "google-beta" {
+    module_name = "blueprints/terraform/terraform-google-memorystore:memcache/v1.0.0"
```
Comment (morgante) [inline modules/memcache/versions.tf:36]:
```suggestion
    module_name = "blueprints/terraform/terraform-google-memorystore:memcache/v2.0.0"
```

**Review by morgante — CHANGES_REQUESTED** [review]

File: versions.tf [hunk 3]
```diff
@@ -15,8 +15,17 @@
  */
 
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 0.13"
```
Comment (bharathkkb) [inline versions.tf:18]: @morgante should we update this to 0.14 since we are now using the 0.14 image?

Comment (morgante) [reply versions.tf:18]: No we shouldn't enforce 0.14 unless we need to.

**Review by morgante — APPROVED** [review]

File: versions.tf (hunk 3 above)
Comment (bharathkkb) [reply versions.tf:18]: ack but we may not be able to detect any breakages to 0.13 automatically for future features added.

**Pull request merged**